app/services/container_dashboard_service.rb

```diff
@@ -86,4 +89,65 @@ def providers
       }
     ]
   end
+
+  def heatmaps
+    # Get latest hourly rollup for each node.
+    node_ids = @ems.container_nodes if @ems
+    metrics = MetricRollup.get_latest_hourly_rollups(ContainerNode.name, node_ids)
+
+    {
+      :nodeCpuUsage    => metrics.collect { |m|
+        avg_cpu = m.cpu_usage_rate_average.round(2)
+        {
+          :id      => m.resource_id,
+          :tooltip => "#{m.resource.name} - #{avg_cpu}%",
+          :value   => avg_cpu / 100.0 # 1% should be 0.01
+        }
+      },
+      :nodeMemoryUsage => metrics.collect { |m|
+        avg_mem = m.mem_usage_absolute_average.round(2)
+        {
+          :id      => m.resource_id,
+          :tooltip => "#{m.resource.name} - #{avg_mem}%",
+          :value   => avg_mem / 100.0 # 1% should be 0.01
+        }
+      }
+    }
+  end
+
+  def node_utilization
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    time_profile = VimPerformanceDaily.default_time_profile(:ext_options => {:tz => @controller.current_user.get_timezone})
+    metrics = VimPerformanceDaily.find_entries(:time_profile => time_profile)
+    metrics = metrics.where(:resource_type => 'ExtManagementSystem', :resource_id => resource_ids)
+    metrics = metrics.order("timestamp")
+
+    used_cpu = Hash.new { |hash, key| hash[key] = 0 }
+    used_mem = used_cpu.dup
+    total_cpu = used_cpu.dup
+    total_mem = used_cpu.dup
+
+    metrics.each do |metric|
+      date = metric.timestamp.strftime("%Y-%m-%d")
+      used_cpu[date] += (metric.derived_vm_numvcpus * metric.cpu_usage_rate_average / 100.0)
+      used_mem[date] += (metric.derived_memory_used)
+      total_cpu[date] += (metric.derived_vm_numvcpus)
```

Comment:
@zeari it shouldn't produce fractions. I've never seen that behavior (both in `metrics` and `metric_rollups`).

Make sure you don't have old entries and use latest master (may be related to #5587)